packages/webapp/src/components/Typography/index.tsx

```diff
@@ -205,3 +207,57 @@ export const Text = ({ children = 'Text', className = '', style, ...props }: Typ
     </p>
   );
 };
+
+export const ExternalLink = ({ children = 'Link', className = '', style, link, ...props }) => {
+  return (
+    <>
+      <div
+        className={clsx(styles.text, className)}
+        style={{ width: 'fit-content', display: 'inline-block', ...style }}
+        {...props}
+      >
+        <Underlined
+          onClick={() => window.open(link, '_blank')}
+          style={{ width: 'fit-content', display: 'inline-block', marginLeft: '3px' }}
+        >
+          {children}
+        </Underlined>
+        <ExternalLinkIcon
+          style={{
+            width: '15px',
+            height: '15px',
+            marginLeft: '2px',
+            position: 'absolute',
+            bottom: '4px',
+          }}
+        />
+      </div>
+    </>
+  );
+};
+
+export const TextWithExternalLink = ({
+  children = 'SubText',
+  className = '',
+  style,
+  link,
+  ...props
+}) => {
+  const { t } = useTranslation(['translation', 'common', 'crop']);
+
+  return (
+    <>
```

Comment:
Could you remove this fragment?

Reply:
this fragment is required for translation